src/lib/components/workflow/filter-bar/dropdown-filter-chip.svelte

```diff
@@ -0,0 +1,553 @@
+                    labelHidden
+                    id="relative-datetime-input"
+                    bind:value={$relativeTimeDuration}
+                    placeholder="00"
+                    error={timeError($relativeTimeDuration)}
+                    class="h-10"
+                    disabled={$timeFormatType !== 'relative' || isNullFilter}
+                  />
+                  <Select
+                    bind:value={$relativeTimeUnit}
+                    id="relative-datetime-unit-input"
+                    label={translate('common.time-unit')}
+                    labelHidden
+                    disabled={$timeFormatType !== 'relative' || isNullFilter}
+                  >
+                    {#each TIME_UNIT_OPTIONS as unit}
+                      <Option value={unit}
+                        >{unit} {translate('common.ago')}</Option
+                      >
+                    {/each}
+                  </Select>
+                </div>
+              </div>
+              <div class="flex flex-col gap-2">
+                <RadioInput
+                  label={translate('common.absolute')}
+                  id="absolute-time"
+                  value="absolute"
+                  name="time-filter-type"
+                  group={timeFormatType}
+                  disabled={isNullFilter}
+                />
+                <div class="ml-6 flex flex-col gap-2">
+                  <DatePicker
+                    label={''}
+                    labelHidden
+                    on:datechange={onStartDateChange}
+                    selected={new Date($startDate)}
+                    todayLabel={translate('common.today')}
+                    closeLabel={translate('common.close')}
+                    clearLabel={translate('common.clear-input-button-label')}
+                    disabled={$timeFormatType !== 'absolute' || isNullFilter}
+                  />
+                  <TimePicker
+                    bind:hour={$startHour}
+                    bind:minute={$startMinute}
+                    bind:second={$startSecond}
+                    twelveHourClock={false}
+                    disabled={$timeFormatType !== 'absolute'}
+                  />
+                </div>
+              </div>
+            {/if}
+            <p
+              class="flex items-center justify-end gap-1 text-sm text-secondary"
+            >
+              <Icon name="clock" aria-hidden="true" />
+              {translate('common.based-on-time-preface')}
+              {selectedTime}
+            </p>
+          </div>
+        {:else if isNumberFilter(localFilter)}
+          <div class="space-y-3">
+            {@render conditionalButtons(numberConditionalOptions)}
+            <Input
+              id={`${controlsId}-number`}
+              label="Value"
+              type="number"
+              placeholder="Enter number..."
+              disabled={isNullFilter}
+              bind:value={localFilter.value}
```

Comment:
Should this have a min value? And should decimals work as well?

```suggestion
              bind:value={localFilter.value}
              min={0}
```

Reply:
Here's the docs from https://docs.temporal.io/search-attribute

Double is backed up by scaled_float Elasticsearch type with scale factor 10000 (4 decimal digits).
Datetime is backed up by date type with milliseconds precision in Elasticsearch 6 and date_nanos type with nanoseconds precision in Elasticsearch 7.
Int is 64-bit integer (long Elasticsearch type).

So I think we want to allow negatives and floats.

Reply:
Added steps="any" to support decimals

<img width="1727" height="951" alt="Screenshot 2025-10-08 at 8 08 13 AM" src="https://github.com/user-attachments/assets/9c2d9ead-5987-4c6d-b685-8b55fd52a67d" />
<img width="1728" height="908" alt="Screenshot 2025-10-08 at 8 08 27 AM" src="https://github.com/user-attachments/assets/641b164d-6feb-4427-98df-38043f6f683e" />
<img width="1728" height="903" alt="Screenshot 2025-10-08 at 8 14 02 AM" src="https://github.com/user-attachments/assets/ded1b6c2-8e53-492e-926a-17bde8fa7c44" />